docs/your_first_grid.md

```diff
@@ -0,0 +1,316 @@
+have to be joined explicitly in the query.
+
+Then you can set up your filter to accordingly:
+
+```yaml
+sylius_grid:
+    grids:
+        app_admin_supplier:
+                ...
+            filters:
+                ...
+                country:
+                    type: string
+                    label: origin
+                    options:
+                        fields: [address.country]
+                    form_options:
+                        type: contains
+```
+
+Default Sorting
+---------------
+
+You can define by which field you want the grid to be sorted and how.
+
+```yaml
+# config/packages/sylius_grid.yaml
+sylius_grid:
+    grids:
+        app_admin_supplier:
+                ...
+            sorting:
+                name: asc
+                ...
+```
+
+Then at the fields level, define that the field can be used for sorting:
+
+```yaml
+# config/packages/sylius_grid.yaml
+sylius_grid:
+    grids:
+        app_admin_supplier:
+            ...
+            fields:
+                name:
+                    type: string
+                    label: sylius.ui.name
+                    sortable: ~
+                ...
+```
+
+If your field is not of a "simple" type, f.i. a twig template with a
+specific path, you get sorting working with the following definition:
+
+```yaml
+# config/packages/sylius_grid.yaml
+sylius_grid:
+    grids:
+        app_admin_supplier:
+            ...
+            fields:
+                ....
+                origin:
+                    type: twig
+                    options:
+                        template: "@App/Grid/Fields/myCountryFlags.html.twig"
+                    path: address.country
+                    label: app.ui.country
+                    sortable: address.country
+                ...
+```
+
+Pagination
+----------
+
+You can limit how many items are visible on each page by providing an
+array of integers into the `limits` parameter. The first element of the
+array will be treated as the default, so by configuring:
+
+```yaml
+# config/packages/sylius_grid.yaml
+sylius_grid:
+    grids:
+        app_admin_supplier:
+                ...
+            limits: [30, 12, 48]
+            ...
+```
+
+you will see thirty suppliers per page, also you will have the
+possibility to change the number of elements to either 12 or 48.
+
+### *Note*
+
+Pagination limits are set by default to 10, 25 and 50 items per page.
+In order to turn it off, configure limits: \~.
+
+Actions Configuration
+---------------------
+
+Next step is adding some actions to the grid: create, update and delete.
+
+### *Note*
+
+There are two types of actions that can be added to a grid: `main`
+which "influence" the whole grid (like adding new objects) and `item`
+which influence one row of the grid (one object) like editing or
+deleting.
+
+```yaml
+# config/packages/sylius_grid.yaml
+sylius_grid:
+    grids:
+        app_admin_supplier:
+                ...
```

Comment:
It is truly a minor thing, but these dots are placed differently in each row. I cannot see any pattern here. IMHO dots should be on the level of current fields. 
```suggestion
            ...
```
or to by 100% correct we should do
```suggestion
            # ...
```